source/YugiAcademy/YuGiOh/views.py

```diff
@@ -89,10 +90,11 @@ def find_or_store_booster_pack(request):
 
 
 def booster_pack_cards(request, booster_pack_id: id):
-    booster_pack = BoosterPack.objects.get(id=booster_pack_id)
+    booster_pack = booster_pack_system.booster_pack_numbered(booster_pack_id)
```

Comment:
Here, it's better booster_pack_with_id... Because if booster pack has a new future field which reference it with a number, could be problematic.

Reply:
What you say is correct. However, when you search for a persistent object by its number, you are referring to its primary key. In some places it is considered a standard.